restore_ports/restore_ports.tcl

```diff
@@ -28,6 +28,8 @@ proc dependenciesForPort {portName variantInfo} {
    if {[llength $portSearchResult] < 2} {
       ui_warn "Skipping $portName (not in the ports tree)"
       return $dependencyList
+   } else {
+       ui_debug "Continuing with $portName"
```

Comment:
This does not really need to be in an `else` block if you want to add more debug output. Just place it after the conditional, which will always return if taken.

Reply:
I think using an `else` makes my intent clearer

Reply:
@raimue what if I change the condition guarding it to an `else if`? (to whatever the Tcl equivalent of "`test -n $portName`" is, that is)

Reply:
For me it would be clearer to see the conditional block that returns for the special case, then follows the normal case. But fine, I don't want to argue hard about such a detail. :smile: